### PR TITLE
[SW-64, SW-65] seed phrase screens

### PR DESCRIPTION
**Pull request by ifaouibadi**

### CODE REVIEW

File: src/popup/router/pages/SeedPhraseDetailsSettings.vue [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+import CheckSuccessCircle from '../../../icons/check-success-circle.svg?vue-component';
+import CopyMixin from '../../../mixins/copy';
+
+export default {
+  components: { Button, CopyOutlined, CheckSuccessCircle },
+  mixins: [CopyMixin],
+  data: () => ({
+    view: 'warning',
+    error: false,
+  }),
```
Comment (CedrikNikita) [inline src/popup/router/pages/SeedPhraseDetailsSettings.vue:66]:
```suggestion
```
Both `error` and `view` is not used in this component.

File: src/popup/router/pages/SeedPhraseDetailsSettings.vue [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+  computed: mapState({
+    mnemonic: 'mnemonic',
+  }),
```
Comment (CedrikNikita) [inline src/popup/router/pages/SeedPhraseDetailsSettings.vue:69]:
```suggestion
  computed: mapState(['mnemonic']),
```

File: src/popup/router/pages/SeedPhraseVerifySettings.vue [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+
+export default {
+  components: {
+    Button,
+    CheckCircle,
+    Alert,
+    Close,
+  },
+  data: () => ({
+    view: 'warning',
```
Comment (CedrikNikita) [inline src/popup/router/pages/SeedPhraseVerifySettings.vue:93]:
```suggestion
```
`view` is not used in this component.

File: src/popup/router/pages/SeedPhraseVerifySettings.vue [hunk 4]
```diff
@@ -0,0 +1,261 @@
...
+    @extend %face-sans-18-regular;
+  }
+
+  .description {
+    color: rgba(variables.$color-white, 0.75);
+    line-height: 20px;
+    text-align: center;
+    padding-bottom: 8px;
+
+    span {
```
Comment (CedrikNikita) [inline src/popup/router/pages/SeedPhraseVerifySettings.vue:148]: https://github.com/aeternity/superhero-wallet/issues/986

**Comment by CedrikNikita** [comment]
seed phrase
For me that it does not fit the screen looks not user friendly

File: src/popup/router/pages/SeedPhraseDetailsSettings.vue [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+
+      &:hover {
+        color: rgba(variables.$color-white, 0.75);
+      }
+    }
+  }
+
+  .button {
+    width: 100%;
+    border-radius: 10px;
```
Comment (konradodo) [inline src/popup/router/pages/SeedPhraseDetailsSettings.vue:152]: $border-radius-interactive

File: src/popup/router/pages/SeedPhraseVerifySettings.vue [hunk 6]
```diff
@@ -0,0 +1,266 @@
...
+      <span class="white-text">
+        {{ $t('pages.seed-phrase-settings.in-correct-order') }}
+      </span>
+    </i18n>
+    <ae-phraser>
+      <ae-badge
+        v-for="(word, index) in mnemonicShuffled"
+        :key="index"
+        :class="{ selected: selectedWordIds.includes(index) }"
+        @click.native="!selectedWordIds.includes(index) && selectedWordIds.push(index)"
```
Comment (konradodo) [inline src/popup/router/pages/SeedPhraseVerifySettings.vue:23]: create method

File: src/popup/router/pages/SeedPhraseVerifySettings.vue [hunk 7]
```diff
@@ -0,0 +1,266 @@
...
+        </ae-badge>
+      </template>
+      <ae-badge
+        v-for="(id, index) in selectedWordIds"
+        v-else
+        :key="id"
+        @click.native="selectedWordIds.splice(index, 1)"
+      >
+        {{ mnemonicShuffled[id] }} <ae-icon name="close" />
+      </ae-badge>
```
Comment (pstachel) [inline src/popup/router/pages/SeedPhraseVerifySettings.vue:51]: You shouldn't use v-for with v-if - it should be wrapped

File: src/popup/router/pages/SeedPhraseVerifySettings.vue [hunk 8]
```diff
@@ -0,0 +1,266 @@
...
+        v-if="showNotification"
+        :class="['notification', { error: hasError }]"
+      >
+        <div class="icon-wrapper">
+          <Alert v-if="hasError" />
+          <CheckCircle v-else />
+        </div>
+        <div class="text">
+          <div v-if="hasError">
+            Seed phrase words are not in the correct order. Please enter your seed phrase again!
```
Comment (pstachel) [inline src/popup/router/pages/SeedPhraseVerifySettings.vue:72]: I think it should be stored in en.json

File: src/popup/router/pages/SeedPhraseVerifySettings.vue [hunk 9]
```diff
@@ -0,0 +1,266 @@
...
+          </div>
+          <div v-else>
+            Seed phrase has been verified!
```
Comment (pstachel) [inline src/popup/router/pages/SeedPhraseVerifySettings.vue:75]: It should be in en.json

**Review by pstachel — CHANGES_REQUESTED** [review]
Also we should stick to naming components with CamelCase not kebab-case - it happens with all `ae-something` components

File: src/popup/router/pages/SeedPhraseVerifySettings.vue [hunk 10]
```diff
@@ -0,0 +1,266 @@
...
+      }
+    }
+  }
+
+  .footer {
+    position: relative;
+
+    .verify-button {
+      width: 100%;
+      border-radius: 10px;
```
Comment (konradodo) [inline src/popup/router/pages/SeedPhraseVerifySettings.vue:222]: variables

**Review by pstachel — APPROVED** [review]

**Pull request merged**